config/database.yml

```diff
@@ -5,21 +5,21 @@
 #   gem 'sqlite3'
 #
 default: &default
-  adapter: sqlite3
+  adapter: postgresql
   pool: <%= ENV.fetch("RAILS_MAX_THREADS") { 5 } %>
   timeout: 5000
 
 development:
   <<: *default
-  database: db/development.sqlite3
+  database: db_development
```

Comment:
You may need to include other options in this config such as `username`, `password`, and the `host`. Also, a more accurate name for the database would be `todo_app_rails_development` instead of `db_development`.

Reply:
While adding the keys mentioned above, make sure you get the database `username` and `password` values from environment variables like so:
`<%= ENV['POSTGRES_USERNAME'] %>` and
`<%= ENV['POSTGRES_PASSWORD']` %>